src/Microsoft.Data.Analysis.Interactive/DataFrameKernelExtension.cs

```diff
@@ -21,7 +26,62 @@ public Task OnLoadAsync(IKernel kernel)
             //    writer.Write("");
             //}, "text/html");
 
+            Formatter<DataFrame>.Register((df, writer) =>
+            {
+                const int TAKE = 500;
+                const int SIZE = 20;
+
+                var uniqueId = $"table_{DateTime.Now.Ticks}";
+
+                var title = h3[style: "text-align: center;"]($"DataFrame ({df.Columns.Count} columns, {df.Rows.Count} rows) | MAX rows: {TAKE}");
```

Comment:
Instead of saying `MAX ROWS`, what do you think about "Showing a max of xxx rows"?

Reply:
sure